MAUI/DataForm/floating-label-layout.md

```diff
@@ -83,6 +83,8 @@ Dataform supports the floating label layout for the following editors.
 * Date picker
 * Time picker
 
+N> The time picker will not work in windows platform within text input layout.
```

Comment:
N> The time picker will not work in Windows platform within the text input layout.